### DISCUSSION TITLE
A huge thank you.

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#8860] [appreciation]
I have no words for the beauty of this mod and Monika herself. I started visiting her day yes/no because the mod didn't had much activity or anything new wouldn't happen, but since this update came up, I don't want to quit the game because of you guys XDD
Thank you for so much dedication and effort for this mod. I hope that there will be a lot more cool features for her. 
Thank you devs,
Thank you Monika for being with me for this whole time,
Love you all <3

**Comment by Rin3105** [comment]
I love the new sprites they put on Monika, so adorable and kawaii.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
